app/src/ui/add-repository/create-repository.tsx

```diff
@@ -121,6 +124,10 @@ export class CreateRepository extends React.Component<
     this.setState({ ...this.state, name })
   }
 
+  private onDescriptionChanged = (description: string) => {
+    this.setState({ ...this.state, description })
```

Comment:
We can just do `this.setState({ description })` now, no need to spread `state` anymore.